src/Command/ImportYamlCommand.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Management.Automation;
+using YamlDotNet;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+using Microsoft.PowerShell.PlatyPS.Model;
+
+namespace Microsoft.PowerShell.PlatyPS
+{
+    /// <summary>
+    /// Get-YamlMetadata reads the YAML header from a markdown file and represents it as a Dictionary object.
+    /// </summary>
+    [Cmdlet(VerbsData.Import, "YamlCommandHelp", HelpUri = "", DefaultParameterSetName = "FromPath")]
```

Comment:
What scenario does the user need this cmdlet?

Reply:
it's a test scenario. We're definitely going to need to import a yamlhelp file to be sure that our translation from markdown to yaml has integrity